app/controllers/talks_controller.rb

```diff
@@ -0,0 +1,11 @@
+class TalksController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @talks = scoped_talks.page(params[:page])
+  end
+
+  def scoped_talks
```

Comment:
Dois pontos aqui:
1. Qual a razão disto estar em um método sendo que é utilizado em 1 lugar só e a regra de negócio é super simples?
2. Se for para ser um método mesmo, o ideal é que ele seja privado

Reply:
Bom ponto, pelo que eu percebi nos controllers do Punch, é usado como padrão um método com padrão `scoped_something`quando se tem um conjunto de dados pertinentes ou ao usuário autenticado ou à função do usuário. Acabei não privando esse método, falha minha. Vou subir essa alteração + o teste solicitado.

Reply:
Refatorado em 4b65e51bbe274485a12cf5fbfa899672d1e431ef e e0856bd1afa747d9ff0b00754828d92d04a3b8b5

Reply:
esse metodo, nesse caso é desnecessário

Reply:
Removido em e8efdb21cc8b6c5b3b301d447c537798bd0bc9d7